persistence/src/main/resources/db/migration/data/integration/V1_12_1__AddStatesToStateTable.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO states (id,name) VALUES (1,'newstate');
```

Comment:
This SQL migration file contains mission critical data, without it Atsy would not work.

In this sense this SQL migration should be placed into the schema folder in order to every environment (production and integration) would be able to built upon it.